### DISCUSSION TITLE
To enable metrics for metacontroller port 9999

### DISCUSSION THREAD

**Discussion by harikrishnang01** [metacontroller/metacontroller#906] [Q&A]
when we trying to integrate metacontroller version 4.10.3 on our systems. we noticed that metrics for the metacontroller service is available at port 9999. but the same is not exposed via statefulset. so is there any plan to expose the metrics 9999 via workload in future

**Comment by grzesuav** [comment]
noft fully following - https://github.com/metacontroller/metacontroller/issues/905 asked in the comment
